templates/infrastructure/account/tfEnvironment/tfRegion/rds/terragrunt.hcl

```diff
@@ -58,15 +59,28 @@ inputs = {
 
   identifier     = "{{ .Spec.identifier }}"
   name           = "{{ .Spec.name }}"
+
+  {{if eq .Spec.engine "postgres"}}
+  // all values correspond to postgres defaults
   engine         = "{{ .Spec.engine }}"
   engine_version = "{{ .Spec.engine_version }}"
+  family       = "{{ .Spec.family }}"
+  major_engine_version       = "postgres13"
+  {{end}}
+
+  {{if eq .Spec.engine "mysql"}}
+  // all values correspond to mysql defaults
+  engine         = "{{ .Spec.engine }}"
+  {{if .Spec.engine_version}}engine_version = "{{ .Spec.engine_version }}"{{else}}engine_version = "8.0.26"{{end}}
```

Comment:
a similar thing needs to be done for postgres too.
can you add that in?

Reply:
Added.

The default values for postgres are already there in *rds module* we only need to add the clause to include the value if they are provided.